README.md

```diff
@@ -713,6 +713,25 @@ timestamps.
 
 [TCurl]: https://github.com/uber/tcurl
 
+## Non filesystem and asynchronous source loading
+
+You can use the alternative asynchronous constructor `Thrift.load()` to load sources from a custom asynchronous function. The function provided will be passed file names and must call the callback function with the source associated with the file name. The callback function provided as third argument to `Thrift.load` will be called when the model is fully loaded or an error occured. This way, you can fetch thrift sources from a server for example. Here is an example :
```

Comment:
nit: extra space between last word and final colon.

Reply:
done